src/com/projectkorra/projectkorra/airbending/flight/FlightMultiAbility.java

```diff
@@ -47,6 +48,9 @@ private static enum FlightMode {
 		SOAR, GLIDE, LEVITATE, ENDING;
 	}
 
+	private Map<FlightMode, Boolean> allowedFlightModes = new HashMap<>();
```

Comment:
If you have a boolean for a value in a map, you may as well just use a Set. And use `Set#contains`